### PR TITLE
ENCD-4171 Reset default audit inherit

### PR DESCRIPTION
**Pull request by hitz**

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -53,7 +53,7 @@ webtest = git https://github.com/Pylons/webtest.git
 WSGIProxy2 = git https://github.com/lrowe/WSGIProxy2.git
 zope.sqlalchemy = git https://github.com/zopefoundation/zope.sqlalchemy.git
 pytest-bdd = git https://github.com/lrowe/pytest-bdd.git branch=allow-any-step-order
-snovault = git https://github.com/ENCODE-DCC/snovault.git rev=1.0.15
+snovault = git https://github.com/ENCODE-DCC/snovault.git branch=SNO-49-change-audit-inherit-default
```
Comment (caseylitton) [inline buildout.cfg]: Reminder to remove after QA completed.

File: src/encoded/types/pipeline.py [hunk 2]
```diff
@@ -177,5 +179,6 @@ class AnalysisStepRun(Item):
     embedded = [
         'analysis_step_version.analysis_step',
     ]
-    # Avoid using reverse links on this object as invalidating a
+    audit_inherit = ['*']
+   # Avoid using reverse links on this object as invalidating a
```
Comment (caseylitton) [inline src/encoded/types/pipeline.py]: Comment has a space removed.

File: src/encoded/types/dataset.py [hunk 3]
```diff
@@ -77,7 +77,7 @@ class Dataset(Item):
     audit_inherit = [
         'original_files',
         'revoked_files',
-        'contributing_files'
+        'contributing_files',
```
Comment (caseylitton) [inline src/encoded/types/dataset.py]: Weird this didn't have a comma before.  I did a mock test, I suspect the prod Dataset.audit_inherit list looks like
```
['original_files', 'revoked_files', 'contributing_filessubmitted_by', 'lab', 'award', 'documents.lab']
```

Comment (hitz) [reply src/encoded/types/dataset.py]: yeah bug of irrelevance

**Pull request merged**